packages/examples/package.json

```diff
@@ -29,15 +29,16 @@
     "demo:inline-completion": "yarn build && node dist/inline-completion.js",
     "demo:simple-chat": "yarn build && node dist/simple-chat.js",
     "demo:simple-completion": "yarn build && node dist/simple-completion.js",
-    "demo:getting-started": "yarn build && node dist/getting-started/index.js",
+    "demo:getting-started": "yarn build && cp src/getting-started/data.json dist/getting-started/ && node dist/getting-started/index.js",
```

Comment:
@NickHeiner I get an error when running `getting-started` since `data.json` is not present. This is a hacky solution. Let me know if there's a better way to fix it.

Reply:
I think this is the best way to do it. Thanks for catching this.